components/brave_wallet/browser/swap_responses.idl

```diff
@@ -111,7 +111,7 @@ namespace swap_responses {
     // The following fields are mandatory for EVM -> any transfers.
     DOMString? from;
     DOMString? to;
-    DOMString? chainId;
+    long? chainId;
     DOMString? value;
```

Comment:
How did this work?
Do/should we use `ConvertAllNumbersToString` converter so all numbers appear as strings?

Reply:
`ConvertAllNumbersToString` appears to be used now in `swap_service.cc` so lgtm.

Reply:
This change from `long?` to `DOMString?` is causing failed to parse error when creating a transaction from a `LiFiQuote` now. Reverting back to `long? chainId` resolves parsing failure. Might be `ConvertAllNumbersToString` not converting for optional types?

Reply:
changing back to from `DOMString` to `long?` causes unit tests to fail

Reply:
Thoughts @supermassive , @onyb ?

Reply:
Fixed with [cb383f6](https://github.com/brave/brave-core/pull/22652/commits/cb383f6f220ad7944a523cf5bdab7912b20fabba)! 🐎